tests/d3d12_descriptors.c

```diff
@@ -6165,7 +6165,7 @@ void test_custom_border_color_limits(void)
         /* NV will fail around 4k unique samplers. */
         if (is_nvidia_device(context.device))
             is_todo = flat_index >= 4000;
-        else if (is_amd_vulkan_device(context.device))
+        else if (is_amd_vulkan_device(context.device) || is_adreno_device(context.device))
```

Comment:
`is_adreno_device()` covers both Turnip and the prop driver. I don't test the prop driver, and I'm not aware of anyone else doing it. The Vulkan HW database lists 4000 for `maxCustomBorderColorSamplers` on one version of the prop driver, so this condition isn't entirely correct for that case.

It might make sense to limit Adreno-specific test tweaks to only Turnip. Let me know, there's only a handful of them currently in the code.

Reply:
I don't particularly care about proprietary qcom given that even DXVK is more or less non-functional on it, I'd be very surprised if vkd3d-proton worked on it at all in any capacity. Don't think we need to complicate things for the time being.